src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx

```diff
@@ -19,13 +18,11 @@ export const SubmissionLayout: React.FC = ({ children }) => {
       <Spacer y={4} />
 
       <AppContainer>
-        <HorizontalPadding>
-          <RouterLink to="/sell" display="block">
-            <ArtsyLogoBlackIcon display="block" />
-          </RouterLink>
+        <RouterLink to="/sell" display="block">
+          <ArtsyLogoBlackIcon display="block" />
+        </RouterLink>
```

Comment:
@dzucconi - should this file be our new layouts pattern?

Reply:
Really, these routes should be using the `LogoOnly` layout. The logo shouldn't be linked to `/sell` it should always go to the homepage. Immediate context should be represented with the `TopContextBar`. 🤷 

Which is to say: I don't think this layout warrants an actual layout; but I do think it should be reconsidered.

Reply:
I agree, it seems weird that a logo would redirect to a page other than the homepage, but Design was specific about this. I only had to remove the horizontal padding.